app/uk/gov/hmrc/trusts/models/TrustRegistrationModels.scala

```diff
@@ -395,41 +437,54 @@ object NonUKType {
 
 case class PropertyLandType(buildingLandName: Option[String],
                             address: Option[AddressType],
-                            valueFull: Long,
-                            valuePrevious: Long)
+                            valueFull: Option[Long],        // now optional with 5MLD
```

Comment:
This change we mandated on the frontend to get around the DES spec being optional. This was to always ensure our frontend provided a value if they have this asset type.
I think it should stay a `Long`.